doc-content/enterprise-only/cases/case-management-clean-case-log-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='case-management-clean-case-log-proc']
+= Removing case logs from the database
+
+Use the `CaseLogCleanupCommand` to cleanup cases, such as canceled cases that lack value and that are using database space. The `CaseLogCleanupCommand` command consists of logic to clean up all or selected cases automatically. By default, the `CaseLogCleanupCommand` is executed once a day but can be configured to run on different intervals.
```

Comment:
> By default, the `CaseLogCleanupCommand` is executed once a day but can be configured to run on different intervals.

Where do we have this information from? I am not aware that it is configured this way by default.

Reply:
Carried over from:

https://access.redhat.com/documentation/en-us/red_hat_jboss_bpm_suite/6.2/html/administration_and_configuration_guide/configuring_logging3

Removed default sentence.